src/wbase/Task.h

```diff
@@ -227,11 +227,14 @@ class Task : public util::CommandForThreadPool {
     bool getFragmentHasSubchunks() const { return _fragmentHasSubchunks; }
     int getSubchunkId() const { return _subchunkId; }
 
-    /// Do not alter the returned value. &&& ??? just return a copy?
+    /// Do not alter the returned value, returns a set of database tables.
+    /// Note: Copying would be safer, but it could be an expensive thing to copy.
+    ///       This only used by ChunkResourceRequest acquire. The set is usually
+    ///       short, but this is called fairly frequently.
     DbTableSet& getDbTbls() { return _dbTbls; }
```

Comment:
Why return the non-`const` reference? It's highly unsafe as it allows a caller to modify a private data member. You comment "Do not alter" is not a protection here 🐱

Reply:
The note basically says it all. I was largely looking for a second opinion. Second opinion says no reference, so make a copy.

Reply:
Actually, returning a constant reference for both of these should be fine. The final `acquire` object makes its own copies.